utils/opencas.conf.5

```diff
@@ -24,7 +24,7 @@ Cache device <DEVICE>
 .br
 Cache mode {wt|wb|wa|pt|wo}
 .br
-Extra fields (optional) ioclass_file=<file>,cleaning_policy=<alru,nop>,promotion_policy=<always,nhit>
+Extra fields (optional) ioclass_file=<file>,cleaning_policy=<alru,nop>,promotion_policy=<always,nhit>,target_failover_state=<active,standby>
```

Comment:
is there any difference between specifying "target_failover_state=active" and not specifying target_failover_state at all?

Reply:
No, there's none. However, all the options are specified in key=value format and I find `target_failover_state=<standby>` strange